### PR TITLE
[Docs] Update example for advanced charts with Vizro-AI

### PR DESCRIPTION
**Pull request by nadijagraca**
## Description
- Update example and relevant images/gif for advanced chart examples with vizro-ai
- Update gifs for chart examples 
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-ai/docs/pages/user-guides/create-advanced-charts.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # How to create advanced charts
 Now, let's explore more advanced visualizations and use Vizro-AI for enhanced formatting.
 
-We'll create an animated bar chart illustrating the population development of each continent over time. Run the code below and look at the result.
+We'll create an animated bar chart illustrating the GDP per Capita of each continent over time. Run the code below and look at the result.
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/create-advanced-charts.md:4]:
```suggestion
We'll create an animated bar chart illustrating the GDP per capita of each continent over time. Run the code below and look at the result.
```

File: vizro-ai/docs/pages/user-guides/create-advanced-charts.md [hunk 2]
```diff
@@ -13,15 +13,16 @@ We'll create an animated bar chart illustrating the population development of ea
...
         fig.show()
         ```
     === "Result"
         [![AnimatedChart1]][AnimatedChart1]
 
-    [AnimatedChart1]: ../../assets/tutorials/chart/animated_bar_chart_1.png
+    [AnimatedChart1]: ../../assets/tutorials/chart/advanced_chart_1.png
 
-Having unveiled our animated bar chart showcasing population development per country, it's clear that crucial details are overshadowed by the legend. Next, we will try to tweak our prompt to group the countries into continents and improve the overall layout.
+Having unveiled our animated map chart showcasing GDP per Capita development per country, it's clear that map area is small, and it is very difficult to differentiate countries.
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/create-advanced-charts.md:24]:
```suggestion
Having unveiled our animated map chart showcasing GDP per capita development per country, it's clear that the map area is small, and it is very difficult to differentiate countries.
```

File: vizro-ai/docs/pages/user-guides/create-advanced-charts.md [hunk 3]
```diff
@@ -33,18 +34,20 @@ Having unveiled our animated bar chart showcasing population development per cou
...
         ```
     === "Result"
         [![AnimatedChart2]][AnimatedChart2]
 
-    [AnimatedChart2]: ../../assets/tutorials/chart/animated_bar_chart_2.png
+    [AnimatedChart2]: ../../assets/tutorials/chart/advanced_chart_2.png
 
 
-By incorporating the directive `Please improve layout`, we've successfully refined our animation and are now able to better interpret our result.
+By incorporating the directive `Increase the width and height of the figure.` and `Show countries on the map.` we've successfully refined our animation and are now able to better interpret our result.
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/create-advanced-charts.md:48]:
```suggestion
By incorporating the directive `Increase the width and height of the figure.` and `Show countries on the map.` we've successfully refined our animation.
```

File: vizro-ai/docs/pages/user-guides/create-advanced-charts.md [hunk 4]
```diff
@@ -33,18 +34,20 @@ Having unveiled our animated bar chart showcasing population development per cou
...
 
-Upon closer inspection, two challenges emerge: the legend overlaps the x-axis and the y-axis range is insufficient to capture the full spectrum of Asia's population development. Let's run the code below to improve the chart.
+Upon closer inspection, the title is too long and the color palette used does not match out needs. We can fix those issue with better and more specific prompting. Let's run the code below to visually improve the chart.
```
Comment (stichbury) [inline vizro-ai/docs/pages/user-guides/create-advanced-charts.md:50]:
```suggestion
Upon closer inspection, the title is too long and the color palette used does not match out needs. We can fix those issues with better and more specific prompting. Let's run the code below to visually improve the chart.
```

**Review by stichbury — APPROVED** [review]
Nice changes, thank you! Just a few minor comments. I think we'll be able to merge this set of updates easily with the changes in #596 since I've hardly touched this page. I'd suggest you merge first, but let's see how the timing works out!

File: vizro-ai/docs/assets/tutorials/chart/animated_advanced_chart.gif
Comment (lingyielia) [inline vizro-ai/docs/assets/tutorials/chart/animated_advanced_chart.gif:1]: Wondering do we have a best practice on gif size limit? So the size is limited to the minimum and also not affecting user experience.

**Review by lingyielia — APPROVED** [review]

File: vizro-ai/docs/assets/tutorials/chart/animated_advanced_chart.gif
Comment (nadijagraca) [reply vizro-ai/docs/assets/tutorials/chart/animated_advanced_chart.gif:1]: Do you think this gif is too large? I can try to adjust the size in snagit.

Comment (stichbury) [reply vizro-ai/docs/assets/tutorials/chart/animated_advanced_chart.gif:1]: @nadijagraca I think the size is OK from what I see in the built docs. I'm personally not a big fan of GIFs as I cannot track them and prefer to use the live dashboard which -- of course -- is coming soon with py.cafe. So let's not sweat this: we will retain the GIFs as they are but soon there will be a link for users to go and use the dashboard directly so if we find the GIF is a nuisance in future (having to regenerate it regularly for example, if we make updates) let's plan to remove it. For now, no action needed!

File: vizro-ai/docs/pages/user-guides/create-advanced-charts.md [hunk 5]
```diff
@@ -33,18 +34,20 @@ Having unveiled our animated bar chart showcasing population development per cou
...
 
-    [AnimatedChart2]: ../../assets/tutorials/chart/animated_bar_chart_2.png
+    [AnimatedChart2]: ../../assets/tutorials/chart/advanced_chart_2.png
 
 
-By incorporating the directive `Please improve layout`, we've successfully refined our animation and are now able to better interpret our result.
+By incorporating the directive `Increase the width and height of the figure.` and `Show countries on the map.` we've successfully refined our animation.
 
-Upon closer inspection, two challenges emerge: the legend overlaps the x-axis and the y-axis range is insufficient to capture the full spectrum of Asia's population development. Let's run the code below to improve the chart.
+Upon closer inspection, the title is too long and the color palette used does not match out needs. We can fix those issues with better and more specific prompting. Let's run the code below to visually improve the chart.
```
Comment (maxschulz-COL) [inline vizro-ai/docs/pages/user-guides/create-advanced-charts.md:50]:
```suggestion
Upon closer inspection, the title is too long and the color palette used does not match our needs. We can fix those issues with better and more specific prompting. Let's run the code below to visually improve the chart.
```

**Review by maxschulz-COL — APPROVED** [review]

**Pull request merged**